model-archiver/model_archiver/arg_parser.py

```diff
@@ -74,6 +74,15 @@ def export_model_args_parser():
                                         'parameter. If --export-path is not specified, the file will be saved in the '
                                         'current working directory. ')
 
+        parser_export.add_argument('--archive-format',
+                                   required=False,
+                                   type=str,
+                                   default="mar",
+                                   choices=["tgz", "mar"],
```

Comment:
Should we change this option to "sagemaker" and "mms" instead of "tgz", "mar"

Reply:
I really think we should go ahead with meaningful values here rather than magical ".mar" or ".tgz". These don't make sense to users on their own without some deeper knowledge of what is the difference. I suggest to go with choices of "cloud", "local" and "aws-sagemaker" with:
- 'cloud' resolving to generate ".mar" file, DEFAULT value
- 'local' resolving to generate only MANIFEST file, without archiving anything
- 'sagemaker' resolving to generate ".tar.gz" file

Also, I suggest to rename this param to "--hosting-at" or something similar. I think that would be more clear to user what this parameter is for (it is not as important to user what file is generated as a result, it is more important for user to know where it meant to be placed for inference to work).

Reply:
I had modified the options to Amazon Sagemaker (I think thats how the service is marketed) and Native MMS. The default value is "Native MMS". I thought these options were very clear to users. Wouldn't "None" be a better option than "Local"? All these artifacts can be hosted locally,  the only difference is in the "archive-format". "None" seems to put across the message to the users users that the tools wouldn't be archiving the model artifacts to any format, more clearly.

Reply:
Ok, these make sense to me. I would only suggest to update "None" to "NoCompression" to be more explicit to user that there will be no archive created as a result of running the tool.